domains/itsrohansahay.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "srohansahay",
+           "email": "f20201697@goa.bits-pilani.ac.in",
+           "discord": "783748496120872960"
+        },
+    
+        "record": {
+            "CNAME": "itsrohansahay"
```

Comment:
invalid CNAME record

Reply:
hey can you tell me what a cname should be? My website is hosted on this[]( https://react-portfolio-vert-nu.vercel.app/)

Reply:
we dont support Vercel

Reply:
but you can use URL tho

Reply:
but i want .is-a.dev domain

Reply:
then use another hosting service, we dont support Vercel and Netlify

Reply:
mm can u pls suggest one hosting service?

Reply:
you can use Github Pages or Cloudflare Pages

Reply:
ok will try'em thanks!